tele_giphy/game/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
-# Create your models here.
+# Keeps tabs on a game token, whether game has started, and ended
+class GameToken(models.Model):
+    game_token = models.CharField(max_length=16)
+    game_active = models.BooleanField(default='False')
+    game_over = models.BooleanField(default='False')
+
+# Can be used for perma users
+class UserToken(models.Model):
+    user_token = models.CharField(max_length=16)
+
+# Keeps track of users in relation to game tokens
+class User(models.Model):
+    username = models.CharField(max_length=60)
+    user_token = models.ForeignKey(UserToken, on_delete=models.CASCADE)
```

Comment:
I think OneToOneField might be better suited for the user_token, since I assume one user will only have one user_token. Maybe for the game_token too, but I'm not sure if you plan on having a user being able to be part of multiple games.